### PR TITLE
FEAT: #158 Template for Email Welcome with image

### LINKED ISSUE #158: [FEAT] EMAIL TEMPLATES > Implement welcome page WITH image
## Description
Using React email package Develop a clean and informative Welcome Email Template. This email is designed to greet new users, confirm their sign-up, and highlight key benefits of the Boilerplate service.

## Acceptance Criteria

- The email template is implemented according to the provided design, WITH an image.
- The template is responsive and displays correctly across various email clients.

### Requirements	
The email template should include the following elements:

- Main heading: "Welcome to Boilerplate!"
- Subheading: "Thanks for signing up"
- Personalized greeting (e.g., "Hi John Doe,")
- Welcome message and brief product description
- Section highlighting key benefits
- Call-to-action button
- Sign-off and company name

### Tests

- Ensure all elements render correctly and match the design.
- Verify responsiveness across various email clients and device sizes.
- Check that personalization fields populate correctly.
- Test the functionality of the CTA button.

###  Tasks

- [ ] Use a React email package to develop the Welcome Email Template based on the provided design and requirements.
- [ ] Implement a responsive layout that works well across different email clients.
- [ ] Set up personalization fields for the recipient's name.
- [ ] Implement the CTA button with proper styling and linking.
- [ ] Ensure all text content is editable for easy customization.
- [ ] Implement tracking parameters for the CTA button (if applicable).

# LINKS

[`FIGMA LINK`](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=7-8277&m=dev).

# IMAGES
image

### PR DESCRIPTION
**Pull request by ruxy1212**
# Description
Built the email template for Welcome page with image.

Closes #158

**Closes #issue_number_here**

# Changes proposed

## What were you told to do?

[FEAT] EMAIL TEMPLATES > Implement welcome page WITH image
https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=402-45684&m=dev

## What did you do?

I developed a clean and informative Welcome Email Template using React email package, designed to greet new users, confirm their sign-up, and highlight key benefits of the Boilerplate service.

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

fnl2

page-m

page-w

### CODE REVIEW

File: src/email/templates/_components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import social1 from "./icons/social-insta";
+import social4 from "./icons/social-lnkdn";
+import social3 from "./icons/social-reddit";
+import social2 from "./icons/social-tiktok";
+import social0 from "./icons/social-x";
+
+export default function Footer() {
+  const icons = [social0, social1, social2, social3, social4];
+  return (
+    <div className="flex justify-center bg-[#e1d6d666] leading-[normal] tracking-[normal]">
```
Comment (taiwonaf) [inline src/email/templates/_components/Footer.tsx:12]: Remove hard coded color

File: src/email/templates/_components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Logo from "./Logo";
+
+export default function Header() {
+  return (
+    <header className="flex w-full items-center justify-center bg-[#e1d6d666]">
```
Comment (taiwonaf) [inline src/email/templates/_components/Header.tsx:5]: Remove hard coded color

File: src/email/templates/welcome/with-image.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                      ))}
+                    </div>
+                  </div>
+                )}
+              </div>
+              <div className="text-center">
+                <Button
+                  href={action_url}
+                  data-testid={BUTTON_TEST_ID}
+                  className="block rounded-[8px] bg-[#F97316] px-10 py-2.5 text-base text-white sm:inline"
```
Comment (taiwonaf) [inline src/email/templates/welcome/with-image.tsx:108]: Remove hard coded color

**Comment by ruxy1212** [comment]
I need help please! React-Email does not support it and cannot import the global.css file
Tailwind variables won't work inside React email. Here's something I found online, a fix from their repo https://github.com/resend/react-email/discussions/1267, however this requires a package.

https://www.npmjs.com/package/@mertasan/tailwindcss-variables

**Pull request closed without merging**